### PR TITLE
August, 17th, 2019 meeting minutes

### PR DESCRIPTION
**Pull request by InessaPawson**
Moving the August, 17th, 2019 NumPy Web Team meeting minutes to the assigned repo.

### CODE REVIEW

**Comment by charris** [comment]
I don't see a problem with just sticking these PRs in, the only thing that might need discussion is the naming conventions we want to follow going forward, we tend to all lower case names. It might be good to get that bit of bike shedding out of the way. @rgommers Thoughts?

File: NumPyWebTeam_meetings/NumPyWebTeam_meeting2019-08-17.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+Time: 9 a.m. Pacific Time
+
+Join Hangouts Meet: https://meet.google.com/trm-gvto-guz
+
+**note: Hangouts works best with Chrome*
+
+Join by phone: +1 980-533-5653 PIN: 455 016 082#
+
+[Slack workspace](https://numpy-team.slack.com)
+
```
Comment (rgommers) [inline NumPyWebTeam_meetings/NumPyWebTeam_meeting2019-08-17.md]: I suggest to remove these details, since they're not relevant for anything going forward.

**Comment by mattip** [comment]
There already is a start of putting these at `status_meetings`, for instance `status_meetings/website_status-2019-09-23.md`. It would be good unify the conventions. I am -1 on mixed case file names, this would as far as I can tell be the first use of them, I don't think it is justified here.

**Comment by rgommers** [comment]
Ah I missed that. Agreed, better to add to that `status_meetings` directory with lowercase naming.

Note that the word "status" is a little odd, we may want to rename at some point.

**Comment by mattip** [comment]
Thanks @InessaPawson

**Pull request merged**